recipes/fire-hpp/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.0":
+    url: "https://github.com/kongaskristjan/fire-hpp/archive/v0.2.tar.gz"
```

Comment:
Hey @toge, the URL is from the previous version

Reply:
@ErniGH 
Sorry, I fixed.